modules/broker-core/src/main/java/org/wso2/broker/core/ExchangeRegistry.java

```diff
@@ -120,7 +102,7 @@ void declareExchange(String exchangeName, Exchange.Type type,
 
         static Exchange newInstance(String exchangeName, Exchange.Type type) throws BrokerException {
             Exchange exchange;
-            switch(type) {
+            switch (type) {
```

Comment:
We didn't get a checkstyle issue for this? Hmm...

Reply:
Nope